week-2/E-conditionals/exercise.js

```diff
@@ -10,7 +10,11 @@ var name = "Daniel";
 var danielsRole = "mentor";
 
 // Write your code here
-
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
+} else {
+  console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
```

Comment:
Both branches of the `if` statement are the same here :thinking: . The example is a bit odd though.